packages/agent/src/invitation/handler.ts

```diff
@@ -115,14 +120,54 @@ export async function resolveOpenId4VciOffer({
   }
 }
 
+export async function popCallback(jwt: {
+  header: Record<string, unknown>
+  payload: Record<string, unknown>
+}) {
+  const header = {
+    ...jwt.header,
+    alg: 'ES256',
+  }
+
+  const payload = jwt.payload
+
+  const toBeSigned = `${TypedArrayEncoder.toBase64URL(
+    TypedArrayEncoder.fromString(JSON.stringify(header))
+  )}.${TypedArrayEncoder.toBase64URL(TypedArrayEncoder.fromString(JSON.stringify(payload)))}`
+  const signature = new Uint8Array(30).fill(42)
+  const jws = `${toBeSigned}.${TypedArrayEncoder.toBase64URL(signature)}`
+  return jws
+}
+
+export function getCreateJwtCallback() {
+  // @ts-ignore
+  return async (_jwtIssuer, jwt) => {
+    const header = {
+      ...jwt.header,
+      alg: 'ES256',
+    }
+
+    const payload = jwt.payload
+
+    const toBeSigned = `${TypedArrayEncoder.toBase64URL(
+      TypedArrayEncoder.fromString(JSON.stringify(header))
+    )}.${TypedArrayEncoder.toBase64URL(TypedArrayEncoder.fromString(JSON.stringify(payload)))}`
+    const signature = await deviceKeyPair.sign(new Uint8Array(TypedArrayEncoder.fromString(toBeSigned)))
+    const jws = `${toBeSigned}.${TypedArrayEncoder.toBase64URL(signature)}`
+    return jws
+  }
```

Comment:
is this b' specfiic? Should we prefix it with that? (just so we know what to remove once we remove this flow again)